src/components/feedback/test/Popover-test.js

```diff
@@ -0,0 +1,316 @@
+import { mount } from 'enzyme';
+import { useRef, useState } from 'preact/hooks';
+
+import Popover, { POPOVER_VIEWPORT_HORIZONTAL_GAP } from '../Popover';
+
+function FakeComponent({ children, ...rest }) {
+  const buttonRef = useRef(null);
+  const [open, setOpen] = useState(false);
+
+  return (
+    <div className="relative">
+      <button
+        data-testid="toggle-button"
+        className="p-2"
+        ref={buttonRef}
+        onClick={() => setOpen(prev => !prev)}
+      >
+        Anchor element
+      </button>
+      <Popover open={open} anchorElementRef={buttonRef} {...rest}>
+        {open &&
+          (children ?? (
+            <>
+              Content of popover
+              <button
+                data-testid="inner-button"
+                onClick={() => setOpen(prev => !prev)}
+              >
+                Focusable element inside popover
+              </button>
+            </>
+          ))}
```

Comment:
The popover should not render its children when it's closed, but a bunch of `Select` tests break if that behavior is changed.

I will tackle that separately, checking what's the best way to fix the Select tests. Perhaps mocking the `Popover` component in that case is the right approach, now that the Popover has its own test.

For now, this condition allows to test it as if it was already behaving like that.

Reply:
Turnso out fixing those tests was easy https://github.com/hypothesis/frontend-shared/pull/1792/files#r1853917255